src/render.js

```diff
@@ -0,0 +1,70 @@
+const { dialog } = require("electron").remote;
+const path = require("path");
+const fs = require("fs");
+
+const viewerElement = document.getElementById("viewer");
+
+const openFileBtn = document.getElementById("open");
+const saveFileBtn = document.getElementById("save");
+
+WebViewer(
+  {
+    path: "../public/lib",
+    initialDoc: "../public/files/webviewer-demo-annotated.pdf",
+  },
+  viewerElement
+).then((instance) => {
+  // Interact with APIs here.
+  // See https://www.pdftron.com/documentation/web for more info
+  instance.setTheme("dark");
+  instance.disableElements(['downloadButton']);
+
+  const { docViewer, annotManager } = instance;
+
+  openFileBtn.onclick = async () => {
+    const file = await dialog.showOpenDialog({
+      properties: ["openFile", "multiSelections"],
+      filters: [
+        { name: "Documents", extentions: ["pdf", "docx", "pptx", "xlsx"] },
```

Comment:
`extentions`

is that a typo?

Reply:
Fixed the typo. Thanks Anthony, does it run on Windows fine?

Reply:
did some quick smoke tests on my end by quickly adding some annotations and trying out printing/downloading.

things looked fine.